src/Parser/Core/Modules/SpellUsable.test.js

```diff
@@ -360,6 +360,15 @@ describe('Core/Modules/SpellUsable', () => {
         });
       }
     });
+    it('A charge spell that has the cooldown of more than a single charge reduced should return the full reduction value', () => {
+      abilitiesMock.getMaxCharges = jest.fn(() => 3);
+      triggerCast(SPELLS.FAKE_SPELL.id);
+      triggerCast(SPELLS.FAKE_SPELL.id);
+      triggerCast(SPELLS.FAKE_SPELL.id);
+      parserMock.currentTimestamp = 6000; //Leaves 1500ms cooldown remainig
```

Comment:
This comment is kinda putting me on the wrong path, how about `leaves 1500ms cooldown remaining of the total 7500ms of the first charge recharging`?